x-pack/legacy/plugins/epm/public/hooks/use_links.tsx

```diff
@@ -53,5 +53,13 @@ export function useLinks() {
       const params = Object.assign({ pkgkey: `${name}-${version}` }, panel ? { panel } : {});
       return appRoot(generatePath(patterns.DETAIL_VIEW, params));
     },
+    toDetailViewRelative: ({ name, version, panel }: DetailParams) => {
```

Comment:
Doesn't have to be in this PR, but I'd love `toDetailView` to call `toDetailViewRelative` instead of having the two functions be nearly identical.

Reply:
i was thinking to not have a new function and pass in a flag that will add appRoot or not (by defaul it would)?